regimes/my/my.go

```diff
@@ -0,0 +1,53 @@
+// Package it provides the Malaysian tax regime.
+package my
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/currency"
+	"github.com/invopop/gobl/i18n"
+	"github.com/invopop/gobl/regimes/common"
+	"github.com/invopop/gobl/tax"
+)
+
+func init() {
+	tax.RegisterRegimeDef(New())
+}
+
+// New instantiates a new Malaysian regime.
+func New() *tax.RegimeDef {
+	return &tax.RegimeDef{
+		Country:   "MY",
+		Currency:  currency.MYR,
+		TaxScheme: tax.CategoryST, // tax/constants.go. Malaysia uses SST — a form of ST in GOBL terms.
```

Comment:
I think it would be good to add a `tax.CategorySST` to represent this unique type in Malaysia.